frontend/src/features/settings/preferences/VideosPollUserSettingsForm.tsx

```diff
@@ -72,12 +104,71 @@ const VideosPollUserSettingsForm = () => {
   return (
     <form onSubmit={handleSubmit}>
       <Grid container spacing={4} direction="column" alignItems="stretch">
-        {/* Generic settings common to all polls */}
+        <Grid item>
+          <Typography variant="h6">
+            {t('pollUserSettingsForm.comparisonPage')}
+          </Typography>
+        </Grid>
+        {/*
+            Comparison UI: weekly collective goal display
+        */}
+        <Grid item>
+          <FormControl fullWidth>
+            <InputLabel
+              id="label__comparison_ui__weekly_collective_goal_display"
+              color="secondary"
+            >
+              {t('pollUserSettingsForm.displayTheTheWeeklyCollectiveGoal')}
+            </InputLabel>
+            <Select
+              size="small"
+              color="secondary"
+              id="comparison_ui__weekly_collective_goal_display"
+              labelId="label__comparison_ui__weekly_collective_goal_display"
+              value={compUiWeeklyColGoalDisplay}
+              label={t(
+                'pollUserSettingsForm.displayTheTheWeeklyCollectiveGoal'
+              )}
+              onChange={changeCompUiDisplayWeeklyColGoal}
+              inputProps={{
+                'data-testid': `${pollName}_weekly_collective_goal_display`,
+              }}
+            >
+              {[
+                {
+                  label: t('pollUserSettingsForm.always'),
+                  value: ComparisonUi_weeklyCollectiveGoalDisplayEnum.ALWAYS,
+                },
+                {
+                  label: t('pollUserSettingsForm.websiteOnly'),
+                  value:
+                    ComparisonUi_weeklyCollectiveGoalDisplayEnum.WEBSITE_ONLY,
+                },
+                {
+                  label: t('pollUserSettingsForm.embeddedOnly'),
+                  value:
+                    ComparisonUi_weeklyCollectiveGoalDisplayEnum.EMBEDDED_ONLY,
+                },
+                {
+                  label: t('pollUserSettingsForm.never'),
+                  value: ComparisonUi_weeklyCollectiveGoalDisplayEnum.NEVER,
+                },
+              ].map((item) => (
```

Comment:
Just a minor note.

List could be defined outside of the component's JSX to make both more readable.

Reply:
Yes, let's do it in the refactor.